gsoc/settings.py

```diff
@@ -242,3 +242,7 @@
 BLOG_MULTISITE = True
 
 INTERNAL_IPS = ('127.0.0.1',)
+
+BOT_NICK = "CommandBot"
```

Comment:
Probably a good idea to have an array of nick's in case one is ever in use.

Reply:
Already using a mechanism to handle that, appending a randomly chosen digit after the name.